android/app/src/main/java/updated/mysterium/vpn/ui/manual/connect/home/HomeActivity.kt

```diff
@@ -20,6 +23,18 @@ class ManualConnectActivity : AppCompatActivity() {
         bindsAction()
     }
 
+    override fun reUseToolbar(toolbarBinding: ToolbarBaseConnectBinding) {
+        changeLeftIcon(R.drawable.icon_menu)
+        if (toolbarBinding.root.parent != null) {
+            (toolbarBinding.root.parent as ViewGroup).removeView(toolbarBinding.root)
+        }
+        binding.manualConnectToolbar.addView(toolbarBinding.root)
+    }
+
+    override fun leftToolbarButtonClicked() {
+        // TODO("Implement burger menu")
+    }
+
     private fun loadData() {
         viewModel.getLocation().observe(this, { result ->
             result.onSuccess {
```

Comment:
Here also should be on fail event handled. For now it can be just Log output, later it can be handled other way

Reply:
Fix

Reply:
Fix